utils/rpms/packaging/rpm_chrootbuild

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+set -uex
+
+echo "$DISTRO_BASE_LOCAL_REPOS"
```

Comment:
(lint) Possible misspelling: DISTRO_BASE_LOCAL_REPOS may not be assigned, but distro_base_local_repos is. [SC2153]